Accept Run Selector txt file as input for `ww-sra-cellranger` (#363)
getwilds/wilds-wdl-library: pipelines/ww-sra-cellranger/README.md

>
> The `ngc_file` parameter is optional. Include it when downloading controlled-access dbGaP data.

### Selecting Samples with SRA Run Selector

`sra_id_file` is the one-accession-per-line text file produced by the **Accession List** button in NCBI's [SRA Run Selector](https://www.ncbi.nlm.nih.gov/Traces/study/). For any study with more than a handful of samples, this is the recommended way to provide inputs. It should look like this:

```
SRR7722937
SRR1039508
SRR13777504
```

> One accession per line, with no header row, commas, or quotes: exactly what the **Accession List** button exports. (Be careful not to grab the adjacent **Metadata** button instead, which downloads a multi-column CSV rather than a bare accession list.)

Reviewer: ```suggestion
One accession per line, with no header row, commas, or quotes: exactly like the file the **Accession List** button exports (under **Select** > **Download** > **Accession List**).
```
Instead of warning what not to click, perhaps give explicit instructions on getting to the right button. Granted, this path may change someday

**Curate your list in Run Selector before running.** This is a manual step, and an important one. Many SRA studies group multiple assay types under a single project: it is common to find single-cell RNA-seq, multiome RNA and ATAC, bulk RNA-seq, and whole-exome runs all under one accession. The SRA metadata does not reliably flag which runs are single-cell. `library_strategy` is `RNA-Seq` for both single-cell and bulk transcriptomic runs, and the only single-cell signal is often a submitter-specific naming convention in the free-text `library_name` field. There is no automated rule that picks the correct Cell Ranger inputs across studies, so the pipeline does not attempt one. Use Run Selector's filters and the per-run library names to select only the runs you intend to feed to Cell Ranger, then export that subset as your accession list.

If a non-single-cell run does slip through, `skip_on_chemistry_failure` is the safety net rather than a substitute for curation. See [Mixed single-cell / non-single-cell input](#mixed-single-cell--non-single-cell-input) below.

> ### Running the Pipeline

Reviewer: ```suggestion
**Filter Run Selector outputs.** Many SRA studies group different assays (e.g. bulk and single-cell) under a single project or sample accession. You may want to use Run Selector's filters before clicking "Accession List." If a non-single-cell run does slip through, `skip_on_chemistry_failure` may catch it. See [Mixed single-cell / non-single-cell input](#mixed-single-cell--non-single-cell-input) below.

```
Consider making this less verbose and jargon-y.

Author: Totally agree on shortening this and removing some jargon, although this suggested change removes the "Running the Pipeline" header. Made a very similar shortening in https://github.com/getwilds/wilds-wdl-library/pull/363/commits/9d1a281973facd948f5fc48fa71264ace74f2f68, great call.

| Parameter | Description | Type | Required? | Default |
|-----------|-------------|------|-----------|---------|

> | `sra_id_file` | Text file of SRA accession IDs, one per line (e.g. the "Accession List" export from [SRA Run Selector](https://www.ncbi.nlm.nih.gov/Traces/study/)). See [Selecting Samples with SRA Run Selector](#selecting-samples-with-sra-run-selector) for details. | File | One of `sra_id_file` / `sra_id_list` | - |

Reviewer: ```suggestion
| `sra_id_file` | Text file of SRA accession IDs, one per line | File | One of `sra_id_file` / `sra_id_list` | - |
```
I think you can keep it short here
